tracer/test/Datadog.Trace.IntegrationTests/StatsTests.cs

```diff
@@ -21,6 +21,24 @@ public class StatsTests
         [Fact]
         public async Task SendStats()
         {
+            await SendStatsHelper(statsComputationEnabled: true, expectStats: true);
+        }
+
+        [Fact]
+        public async Task SendsStatsOnlyAfterSpansAreFinished_TS008()
```

Comment:
Out of interest, what are the `_TS008` and `_TS010` suffixes on these test methods? Is it because they're copied from other repos?

Reply:
I asked the same question. Here's Zach's reply: 

> To define the cross-tracer stats computation tests we gave an identifier for each test case, so this matches up with the system tests identifier, so we have a test in our own repo that can be used for debugging fail system tests